README.MD

```diff
@@ -45,11 +55,7 @@ then import the banner into your code:
 
 ##### From a CDN
 
-You can include the minified script directly from unpkg cdn:
-
-```html
-<script src="//unpkg.com/@beyonk/gdpr-cookie-consent-banner@3.1.0/dist/browser/bundle.min.js"></script>
-```
+No CDN link as of now. Please see the methods above and below on how to either include the esm package or a local file.
```

Comment:
We can probably put the original line back here and remove the `version` from the URL

Reply:
Yes, sure. I'll revert that and remove the message on top as soon as this is merged.